js/posts/postManagement/searchPost/searchPosts.mjs

```diff
@@ -0,0 +1,34 @@
+import { renderPosts } from "../../renderPosts.mjs";
+import { getSearchTerms } from "./getTerms.mjs";
+import { postNotFoundError } from "./postNotFound.mjs";
+
+export async function searchPosts(postData, url) {
+  const postsArray = await postData(url);
+  const searchForm = document.querySelector("input#searchInput");
+  const postContainer = document.querySelector(".write-post");
+  //
+  searchForm.addEventListener("input", onSearch);
+  //
+  function onSearch(e) {
+    e.preventDefault();
+    const searchTerm = e.target.value;
+    const term = searchTerm.toLowerCase();
+    if (!term.length) {
+      postContainer.innerHTML = "";
```

Comment:
This will eventually lead to a crash. It's better to use another approach despite how convenient this is.

```js
HTMLElement.prototype.clear = function() {
  while (this.firstChild) {
    this.removeChild(this.firstChild);
  }
}
```

Reply:
```js
postContainer.clear()
```

Reply:
Ye i tried this earlier but im getting the same message now as i did previously:
![bilde](https://user-images.githubusercontent.com/96174153/222885765-69045aff-4e1f-4994-9c05-52a7f51671e5.png)